### DISCUSSION TITLE
Issue Training VITS on a custom Dataset

### DISCUSSION THREAD

**Discussion by rob1392** [coqui-ai/TTS#1704] [General Q&A]
I have created a custom dataset from a variety of publicly accessible audio recordings and am having trouble training VITS to generate intelligible speech. The dataset has multiple speaker totaling ~200 hours with the distribution of speaker times' looking like this:

![all_speakers](https://user-images.githubusercontent.com/108382968/176347637-62d6111b-36dd-4800-9f5c-785f8ec6dcc8.png)

I also tried just the top 6 speakers representing ~85 hours.

I have used all of the same parameters from here: https://github.com/coqui-ai/TTS/blob/dev/recipes/vctk/vits/train_vits.py with the main difference being I am training on 8 V100 GPUs.

In order to create the dataset I followed the advice of the HiFiTTS paper: https://arxiv.org/abs/2104.01497 using CTC to align speech and text and filtering out everything that had a SNR of at least 20 dB.

The audios when I spot checked them all sound good and the distribution of audio lengths also seems good looking like this:
![audio_length](https://user-images.githubusercontent.com/108382968/176348415-2860260d-a320-4719-9ad9-fedbdd3e7872.png)

I have trained these models for a relatively short time but it seems the asymptotes are not reaching the levels of other successfully trained models like VITS on VCTK (which I provide as reference) so my suspicion is that more training time is not the answer and the problem lies elsewhere.

I have attached my tensorboard charts here in the hopes that someone can help me out.

The blue lines are the baseline VITS trained on VCTK and the red lines are the same VITS params trained on my custom dataset.
To my eyes it seems like the discriminator is "winning" a lot more in the custom instance and so maybe there is some tweaking that could be done there?

![train_stats_page_1](https://user-images.githubusercontent.com/108382968/176348570-6de570ae-2ab5-42de-a891-c10e0453fc70.png)
![train_stats_page_2](https://user-images.githubusercontent.com/108382968/176348576-10aa6da5-df6f-40cb-9a1b-c71e3341d7fd.png)

![eval_stats_page_1](https://user-images.githubusercontent.com/108382968/176348606-8cf61fc4-2345-4f88-b226-5db8cd38861a.png)
![eval_stats_page_2](https://user-images.githubusercontent.com/108382968/176348617-341f82c7-11a7-4475-96af-646fa83ee323.png)

![eval_imgs](https://user-images.githubusercontent.com/108382968/176348627-0cedfe4f-bc5d-451f-9dd3-165cabc7f9e0.png)

Please let me know if I am missing any information and thanks for the help!

**Comment by neurlang** [comment]
what is your audio sample rate?

**Comment by xanguera** [comment]
Dear @rob1392 did you manage to find what the problem was?
Thank you
